lib/mbta.js

```diff
@@ -1,11 +1,28 @@
 'use strict';
 
 // Code here.
+let lines = {
+'Red': ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard',
+  'Porter', 'Davis', 'Alewife'],
+'Green': ['Government Center', 'Park Street', 'Boylston', 'Arlington',
+  'Copley', 'Hynes', 'Kenmore'],
+'Orange': ['North Station', 'Haymarket', 'Park Street', 'State',
+  'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills']
+};
 
```

Comment:
Great data structure.  Next time use const for this though as it will not change.